### PR TITLE
docs: Japanese localization

### PR DESCRIPTION
**Pull request by LucasMRC**
In this PR, we add japanese translations for:
* docs/sources/_index.ja.md
* docs/sources/get-started/overview.ja.md

**Checklist**
- [ ] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)
- [ ] Documentation added
- [ ] Tests updated
- [ ] Title matches the required conventional commits format, see [here](https://www.conventionalcommits.org/en/v1.0.0/)
  - **Note** that Promtail is considered to be feature complete, and future development for logs collection will be in [Grafana Alloy](https://github.com/grafana/alloy). As such, `feat` PRs are unlikely to be accepted unless a case can be made for the feature actually being a bug fix to existing behavior.
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/setup/upgrade/_index.md`
- [ ] For Helm chart changes bump the Helm chart version in `production/helm/loki/Chart.yaml` and update `production/helm/loki/CHANGELOG.md` and `production/helm/loki/README.md`. [Example PR](https://github.com/grafana/loki/commit/d10549e3ece02120974929894ee333d07755d213)
- [ ] If the change is deprecating or removing a configuration option, update the `deprecated-config.yaml` and `deleted-config.yaml` files respectively in the `tools/deprecated-config-checker` directory. [Example PR](https://github.com/grafana/loki/pull/10840/commits/0d4416a4b03739583349934b96f272fb4f685d15)

### CODE REVIEW

**Comment by JStickler** [comment]
Note to Loki Engineering, please DO NOT MERGE this PR without docs approval.  The Docs team is discussing the details about how we're going to handle localized documentation (in a Slack thread at the moment, but there may be further discussions).

File: docs/sources/get-started/overview.ja.md
Comment (JStickler) [inline docs/sources/get-started/overview.ja.md:20]: Google Translate says that this line reads "Agent - An agent or client, such as Promtail or Grafana Agent that ship with Loki."  Please change this to reference Alloy, as the Loki documentation has replaced references to Grafana Agent with Grafana Alloy.

**Review by GrafanaWriter — CHANGES_REQUESTED** [review]
@LucasMRC - can you please close this PR? 

I believe we have some crossed wires here; this PR is already stale, it's not clear that we have a plan for maintenance of versioned documentation that is translated, nor is there yet a design to support its inclusion.

**Comment by JStickler** [comment]
Closing per docs management request until there is a Localization plan in place.

**Pull request closed without merging**
